test/drivers/test_suites/syscall_exit_suite/execveat_x.cpp

```diff
@@ -211,7 +211,7 @@ TEST(SyscallExit, execveatX_correct_exit)
 
 	evt_test->disable_capture();
 
-#if defined(__s390x__) || defined(__riscv)
+#if defined(__s390x__) || defined(__riscv) || defined(__powerpc__)
```

Comment:
In this file you are using `__powerpc__` in place of `__ppc64le__`. Is it fine?

Reply:
See userspace/libscap/engine/gvisor/parsers.cpp changes.

Reply:
Yes. `__ppc64le__` does not work.

Reply:
So we need to use `__powerpc__` in userspace/libscap/engine/gvisor/parsers.cpp ?

Reply:
What about `__powerpc64__` (looking at this: https://reviews.llvm.org/D137513)?

Reply:
RIght. I will update it. Honestly, it is a bit confusing with different nomenclatures.

Reply:
Agree...

Reply:
Alternatively, I can use `__PPC64__`
https://www.ibm.com/docs/vi/xl-c-and-cpp-linux/16.1.0?topic=macros-related-platform

Reply:
I'd go with the lowercase one if that works :D

Reply:
That works. :D. I guess `__powerpc64__` makes more sense.